pkg/api/core/v1/models/models.go

```diff
@@ -24,8 +24,8 @@ type ServiceRef struct {
 
 // ServiceResponse represents the data of a single service instance
 type ServiceResponse struct {
-	Meta      ServiceRef `json:"meta"`
-	BoundApps []string   `json:"boundapps"`
+	Meta ServiceRef          `json:"meta"`
+	Spec ServiceShowResponse `json:"spec"`
```

Comment:
Minor comment, the response to apps contains spec like details in a `configuration` property. For conformity could they use the same name?

App Response
```
{
        "configuration": {
            "environment": {},
            "instances": 1,
            "routes": [
                "from-archive.192.168.0.2.omg.howdoi.website"
            ],
            "services": [
                "db-service-2"
            ]
        },
        "deployment": {
...
        },
        "meta": {
...
        },
        "status": "running",
        "statusmessage": ""
    }
```

Reply:
Ah, you want current `Spec` to be called `Configuration`, right ? If so, trivial change, no problem.

Reply:
@andreas-kupries :+1:

Reply:
commit 5853e2ec8dcda5a7a150f41d1d494db5ee39f3ea